README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Modrinth](https://img.shields.io/modrinth/dt/nvidium?logo=modrinth)](https://modrinth.com/mod/nvidium)
 
-Nvidium is an alternate rendering backing for sodium, it uses cutting edge nvidia features to render huge amounts of
+Nvidium is a replacement rendering backend for Sodium, using cutting edge NVIDIA-exclusive OpenGL extensions to render huge amounts of
```

Comment:
- If you want to stay as close as possible to the original wording, I think she maybe meant "alternative"?
- I would probably leave it at "NVIDIA features", as users might not know what an OpenGL extension is.